README.rst

```diff
@@ -15,9 +15,9 @@
     See the License for the specific language governing permissions and
     limitations under the License.
 
-==============================
- Renku CLI and SDK for Python
-==============================
+=================================
+ Renku Python library and Service
```

Comment:
What happened to CLI?

Reply:
I was trying to not be too verbose. 

"Renku Python Library, CLI, and Service"?